tests/integration/errors/test_E_WARNING.php8.php

```diff
@@ -31,7 +31,7 @@
     [
       "?? when",
       "OtherTransaction/php__FILE__",
-      "include(): Failed opening 'abc.php' for inclusion (include_path='.:')",
+      "/include\\(\\): Failed opening 'abc.php' for inclusion \\(include_path='.:.*'\\)/",
```

Comment:
adding the '/' as a prefix and suffix makes the integration runner read the string as regex.

adding '\\' before the "()" escapes the "()" for the regex string. Double backslash is needed because first the string is parsed through the go json library, which will condense the escaped '\\' to just a '\'

Regex is needed to add a ".*" in the include_path, as the dev environment has an additional path on its include_path